connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -267,6 +267,18 @@ public String memberId() {
         return JoinGroupRequest.UNKNOWN_MEMBER_ID;
     }
 
+    @Override
+    protected void handlePollTimeoutExpiry() {
+        log.warn("worker poll timeout has expired. This means the time between subsequent calls to poll() " +
+            "in DistributedHerder tick() method was longer than the configured rebalance.timeout.ms. " +
+            "If you see this happening consistently, then it can be addressed by either adding more workers " +
+            "to the connect cluster or by increasing the rebalance.timeout.ms configuration value. Please note that " +
```

Comment:
The bit about `then it can be addressed by either adding more workers`, is something which has been experienced in certain cases when there are too many tasks to start which leads to poll timeout. I can remove it if it's not deemed necessary here.

Reply:
I think this is decent advice when requests are small and can be distributed around the cluster, but as REST requests are rather infrequent, I think this is the minority of cases.

I think most often this timeout is going to be triggered by an excessively slow connector start, stop, or validation. In those cases, adding more workers does nothing but move the error to a different worker. I think we can keep the "adding more workers" comment, if we include another piece of advice for debugging excessively blocking tasks. If we don't have that other piece of advice, then advising users to add workers is misleading.